code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/react-in-jsx-scope */
+import './reset.css';
+import './styles.css';
+import './dark-theme.css';
```

Comment:
Styles should be imported in the very bottom of import section, below custom components import